src/shared/js/utils/functions.js

```diff
@@ -441,6 +441,36 @@ const tablerSvg = (pathName, id, classNames) => {
                 <path d="M14 15v2a1 1 0 0 1 -1 1h-7l-3 3v-10a1 1 0 0 1 1 -1h2" />
             </svg>`;
 
+        case "vanity":
```

Comment:
Not perfect

Reply:
No but close enough! I would keep that version :)